Views/all_tasks.phtml

```diff
@@ -0,0 +1,70 @@
+<form action="/task/deleteTask" method="post">
+    <table border="1">
+        <tr>
+            <td>id</td>
+            <td>Время создания</td>
+            <td>Создал</td>
+            <td>Исполнитель</td>
+            <td>Задание</td>
+            <td>статус</td>
+            <td>Время начала</td>
+            <td>Время окончания</td>
+            <td>Комментарий</td>
+            <td>Подтверждено кем</td>
+            <td>Изображение</td>
+            <td>
+                <button type="submit">Удалить</button>
+            </td>
+        </tr>
+        <?php
+        foreach ($_SESSION["allTasks"] as $tasks) {
+            echo "<tr>";
```

Comment:
use html tags instead of <?php echo ?>
if you refactor it to work with short syntax you can make it like
```
<?php foreach ($result as $index => $value): ?>
<tr>
<?php endforeach; ?>
```